src/error-handling/result.md

```diff
@@ -0,0 +1,77 @@
+---
+minutes: 5
```

Comment:
Since this is review and doesn't go into `?` it probably needs less time than this (but it's just an estimate so _shrug_)

Reply:
I think 5 minutes is pretty accurate. When I get to this point I don't just talk about `Result` but about Rust's error handling strategy more generally.